File: giovanna-britto/2652/sum_multiples.py
class Solution:
    def sumOfMultiples(self, n: int) -> int:
        total = 0

        for i in range(n +1):
            if (i % 3 == 0 or i % 5 == 0 or i % 7 == 0):
                total += i

        return total

File: GabrielScarpelinDiniz/2652/sumMultiples.rs
impl Solution {
    pub fn sum_of_multiples(n: i32) -> i32 {
        let mut num = 1;
        let mut sum = 0;
        while num <= n {
            if num % 3 == 0 || num % 5 == 0 || num % 7 == 0 {
                sum = sum + num;
            }
            num += 1;
        }
        sum
    }
}
